README.md

```diff
@@ -395,6 +397,36 @@ If you want to use [Relay](https://facebook.github.io/relay/) together with this
 
 Please read our [guidelines](.github/CONTRIBUTING.md).
 
+## Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](.github/CONTRIBUTING.md)].
```

Comment:
Add an addition linebreak after this line. 

Also, there are square brackets around the link, I would them to be removed if that's ok with you.